include/swift/SIL/SILModule.h

```diff
@@ -323,7 +323,7 @@ class SILModule {
   }
 
   /// Returns true if everything in this SILModule is being serialized.
-  bool isWholeModuleSerialized() const { return Options.SILSerializeAll; }
+  bool isWholeModuleSerialized() const { return false; }
```

Comment:
Are you going to remove this, and ResilienceStrategy::Fragile, etc?

Reply:
Yes, I'm going to remove it as well. There is no need for it anymore.

Reply:
Also makeModuleFragile() in SILGen